app/(app)/[speaker].tsx

```diff
@@ -1,28 +1,212 @@
-          title: `Speaker ${name}`,
-          headerTitleAlign: 'center',
+          title: type === 'speaker' ? 'Speaker' : 'Organizer',
+          headerTitleAlign: 'left',
+          headerLeft: () => (
+            <AntDesign name="arrowleft" size={24} color={colors.whiteConstant} onPress={() => router.back()} />
+          ),
         }}
       />
-      <StyledText>speaker id {name}</StyledText>
+      <View style={styles.main}>
+        <ImageBackground
+          source={
+            dark ? require('../../assets/images/speaker-dark.png') : require('../../assets/images/speaker-light.png')
+          }
+          style={styles.banner}
+          resizeMode="cover"
+        />
+
+        <View style={styles.centered}>
+          <View style={styles.avatar}>
+            <Image
+              source={details?.image}
+              style={[styles.image, { borderColor: colors.tint }]}
+              contentFit="contain"
+              transition={1000}
+            />
+          </View>
+          <View style={styles.info}>
+            <View style={styles.row}>
+              <FontAwesome5 name="android" size={18} color={colors.tertiary} />
+              <Space size={5} horizontal />
+              <StyledText size="sm" font="regular" style={{ color: colors.tertiary }}>
+                {type === 'speaker' ? 'Speaker' : 'Organizer'}
+              </StyledText>
+            </View>
+            <StyledText size="lg" font="bold" variant="link">
+              {name}
+            </StyledText>
+
+            <Space size={10} />
+            <View style={styles.taglineContainer}>
+              <StyledText size="base" font="regular" style={[styles.text, { color: colors.textLight }]}>
+                {details?.tagline}
+              </StyledText>
+            </View>
+          </View>
+        </View>
+
+        <View style={styles.content}>
+          <StyledText size="lg" font="bold" variant="link">
+            Bio
+          </StyledText>
+
+          <Space size={8} />
+
+          <StyledText size="base" font="light" style={styles.contentText}>
+            {details?.bio}
+          </StyledText>
+        </View>
+
+        <Space size={16} />
+
+        <View style={[styles.socialLink, { borderTopColor: colors.borderColorSecondary }]}>
+          <Row>
+            <StyledText font="medium">Twitter Handle</StyledText>
+
+            <Pressable
+              style={[styles.button, { borderColor: colors.primary, backgroundColor: colors.background }]}
+              onPress={() => handleTwitterProfile(details?.twitter_handle)}
+            >
+              <FontAwesome5 name="twitter" size={20} color={colors.primary} />
+              <Space size={4} horizontal />
+              <StyledText font="medium" style={{ color: colors.primary }}>
+                {details?.twitter_handle ? getTwitterHandle(details?.twitter_handle) : 'N/A'}
+              </StyledText>
+            </Pressable>
+          </Row>
+        </View>
```

Comment:
![photo_2023-10-27_10-38-38](https://github.com/droidconKE/droidconKE2023ReactNative/assets/28436092/e62baedf-8399-432d-8372-eb17481f16c0)

The social share row should be at the bottom based on the design. I have been trying to make it work to no avail.

Reply:
I've fixed it